crates/provider-sqldb-postgres/wit/deps/postgres/types.wit

```diff
@@ -0,0 +1,299 @@
+package wasmcloud:postgres@0.1.0-draft;
+
+/// Types used by components and providers of a SQLDB Postgres interface
+interface types {
+
+  /// Errors that occur while executing queries
+  variant query-error {
+    /// Unknown/invalid query parameters
+    invalid-params(string),
+    /// Invalid/malformed query
+    invalid-query(string),
+    /// A completely unexpected error, specific to executing queries
+    unexpected(string),
```

Comment:
Nit: Should we have a "No rows" error like what `database/sql` has in Go?

Reply:
I don't think so -- no rows being returned is not necessarily a bug...